pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -368,6 +383,7 @@ func (i *CommandInitOption) genCerts() error {
 func (i *CommandInitOption) prepareCRD() error {
 	if strings.HasPrefix(i.CRDs, "http") {
 		filename := i.KarmadaDataPath + "/" + path.Base(i.CRDs)
+		fmt.Println(i.CRDs)
```

Comment:
```suggestion
```

Reply:
The issues mentioned above have been resolved, and I look forward to your further guidance 🌼

Reply:
The issues mentioned above have been resolved, and I look forward to your further guidance 🌼
@liangyuanpeng